lib/utils/job-utils/ipmi-parser.js

```diff
@@ -163,8 +162,11 @@ function parseSensorsDataFactory(assert, _) {
                     if (key.search(/sensorType/) >= 0) {
                         var sensorTypeKey = key.slice(0, 'sensorType'.length);
                         var sdrTypeValue = key.slice('sensorType'.length, key.length);
+                        var match = value.match(/(.+) (\(0x.+\))?/);
                         key = sensorTypeKey;
-                        value = value.match(/(.+) \(0x.+\)/)[1].trim();
+                        if (match) {
+                            value = match[1].trim();
+                        }
                         sdrObj.sdrType = sdrTypeValue;
```

Comment:
suggest to change "Analog" to "Threshold" here, since sdr alert job will use this keyword. https://github.com/RackHD/on-tasks/blob/master/lib/jobs/ipmi-sdr-alert-job.js#L186